jso/apis/src/main/java/org/teavm/jso/dom/events/Touch.java

```diff
@@ -0,0 +1,62 @@
+/*
+ *  Copyright 2014 Alexey Andreev.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.jso.dom.events;
+
+import org.teavm.jso.JSObject;
+import org.teavm.jso.JSProperty;
+
+// From https://developer.mozilla.org/en-US/docs/Web/API/Touch
+
+public interface Touch extends JSObject {
+
+    @JSProperty
+    long getIdentifier();
```

Comment:
This won't work. JavaScript does not provide int64 type (or at least, did not provide by the time touch event spec was written). I'd replaced this with `int`, but you should test on real use cases if `int` fits well or you need `double`.